### PR TITLE
feat:added ens support in connect button

### PR DESCRIPTION
**Pull request by 3scava1i3r**
---
name: 'feat:added ens support in connect button component'
about: Pr to fix issue #698 
---

### Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/main/SECURITY.md).
- [x] My code is conform the [code style](https://github.com/web3ui/web3uikit/blob/master/CONTRIBUTE.md)
- [x] I have made corresponding changes to the documentation
- [x] I have updated Typescript definitions when needed

### Issue Description

Related issue: #698 

### Solution Description
final image is below 
ss

### CODE REVIEW

**Review by BillyG83 — APPROVED** [review]
wow @3scava1i3r nice work

**Comment by BillyG83** [comment]
hey @3scava1i3r this one failed because of 

```
(typescript) Error: /home/runner/work/web3uikit/web3uikit/src/components/ConnectButton/ConnectButton.tsx(42,19): semantic error TS6133: 'isLoading' is declared but its value is never read.
```

could you remove this please =]

File: src/components/ConnectButton/ConnectButton.tsx [hunk 1]
```diff
@@ -39,6 +39,7 @@ const ConnectButton: React.FC<ConnectButtonProps> = ({
         Moralis,
     } = useMoralis();
 
+    const { name, isLoading, error } = useEnsAddress(String(account));
     const [isConnectModalOpen, setIsConnectModalOpen] = useState(false);
```
Comment (AbhinavMV) [inline src/components/ConnectButton/ConnectButton.tsx:43]: please remove isLoading and error as it is not being used anywhere

**Comment by AbhinavMV** [comment]
Nice job @3scava1i3r

**Pull request merged**